src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php

```diff
@@ -130,16 +128,16 @@
     $sFormTabsContent .= '</div></div>';
 }
 
-/** add handling of ENTER key to trigger the save button if we have a form with only one text field */
-if (!empty($sLastTextFieldName) && 1 == $iTextFieldCount) {
+/** add handling of ENTER key to trigger the save button if we have a form with a text field as last field */
```

Comment:
(It's not a function documentation ( /**) so the comment should be a normal one (//).)